RELEASE-NOTES.md

```diff
@@ -0,0 +1,281 @@
+### 0.9.0
+#### Signing Service and Middleman
+- Signing Service and Middleman are now fully implemented but disabled by default (#673, #633, #616, #629, #618, #625, #632).
+- Concent can now be configured to use Signing Service instead of signing transaction by itself (#635).
```

Comment:
Maybe, like for `Bugfix:`, we should add a prefix for other types, ie. `Feature: ` or `Improvement: `

Reply:
I think it's clear from context. I only added it for bugs not to constantly write "fixes bug xxx".

What do others think?